src/index.html

```diff
@@ -22,6 +22,373 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          class="logo"
+          src="/src/images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list-items">
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link is-active"
+            >
+              Apple
+            </a>
```

Comment:
The class 'is-active' suggests that this is the current active page. If 'Apple' is not the current page, this class should be removed. If it is the current page, the href attribute should be set to '#' to indicate that it is the current page.